README.md

```diff
@@ -63,6 +64,27 @@ sudo apt-get install tezos-baker-007-psdelph1
 ```
 Once you install such packages the commands `tezos-*` will be available.
 
+<a name="raspberry"></a>
+### Ubuntu packages on Raspberry Pi OS
+
+If you have a Raspberry Pi running the 64bit version of the official OS, you can
+use the Lauchpad PPA to install `tezos-*` executables on it as well.
+
+You can add the PPA using:
+```
+# Intall software properties commons
+sudo apt-get install software-properties-common
+# Add PPA with Tezos binaries
+sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
```

Comment:
this doesn't appear to be a valid key: command fails and [lookup via web interface](https://keyserver.ubuntu.com/pks/lookup?search=37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB&fingerprint=on&op=index) also returns "not found"

Reply:
@itkach I wonder if it was a momentary failure, could you try again?
I checked and the key seems to be correct.

You can see that's the signing key from the [lauchpad page](https://launchpad.net/~serokell/+archive/ubuntu/tezos) (under the `Technical details about this PPA`).

The ubuntu keyserver web interface seems to require the `0x` prefix, with that the [key can be found](https://keyserver.ubuntu.com/pks/lookup?search=0x37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB&fingerprint=on&op=index).

I also checked again on my Pi and was able to `apt-get update` and `upgrade` to `v.9`, as well as delete and re-import the key with the command above.

Reply:
advice here helped: https://www.laurivan.com/apt-key-adv-fix-server-indicated-a-failure-on-raspberry-pi/ - key was successfully imported after changing keyserver url to `hkp://keyserver.ubuntu.com:80` (specify port 80 explicitly)

Reply:
:ok_hand: added the port explicitly now. Oddly both with and without it seem to work for me.